bridge/dd_register_optional_deps_autoloader.php

```diff
@@ -13,19 +13,12 @@ class OptionalDepsAutoloader
      * @var array
      */
     private static $autoloaderMapping = [
-        "DDTrace\\Integrations\\ZendFramework\\V1\\TraceRequest" => 'DDTrace/Integrations/ZendFramework/V1/TraceRequest.php',
```

Comment:
This was added back.

Reply:
I suppose we don't have a test case for this manual stuff, do we?

Reply:
Apparently not; but considering the docs advise to manually move the files over, we should be good in this specific case. But overall the ZF1 manual instrumentation needs to be audited. I created a card in the backlog for that.